usecases/objects/references_test.go

```diff
@@ -50,7 +50,7 @@ func Test_ReferencesAddDeprecated(t *testing.T) {
 		}, nil)
 		expectedRefProperty := "hasAnimals"
 		source := crossref.NewSource(schema.ClassName(cls), schema.PropertyName(expectedRefProperty), id)
-		target := crossref.New("localhost", "", "d18c8e5e-a339-4c15-8af6-56b0cfe33ce7")
+		target := crossref.New("localhost", "Animal", "d18c8e5e-a339-4c15-8af6-56b0cfe33ce7")
```

Comment:
can we also preserve the tests that test the cross refs without the class as it used to be?